Swifter.xcodeproj/project.pbxproj

```diff
@@ -775,6 +779,7 @@
 					"$(inherited)",
 				);
 				INFOPLIST_FILE = SwifterDemoiOS/Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 13.0;
```

Comment:
Don't change the deployment target; look at what I did in f121d096cf3f1b8bdf9516315d70398ec4cd7bf1 to support SceneDelegate (i.e. Implement both URL opening mechanisms in UIApplicationDelegate and UISceneDelegate so that iOS 13 and above will use scene implementation and anyone lower will fall back to application implementation).

Reply:
Thanks. I merged master branch. So, supported older iOS in this branch.